ultravox/tools/ds_tool/ds_tool.py

```diff
@@ -131,55 +173,57 @@ class DatasetToolArgs:
     )
 
     def __post_init__(self):
-        assert (
-            not self.upload_split or self.dataset_split
-        ), "Must specify dataset_split when using upload_split"
+        assert self.dataset_subset, "dataset_subset must be specified"
```

Comment:
I don't think this condition is always true?

Reply:
you are right.